libs/brig-types/test/unit/Test/Brig/Roundtrip.hs

```diff
@@ -55,3 +56,14 @@ testRoundTripWithSwagger = testProperty msg (trip .&&. scm)
             validatePrettyToJSON v
         )
         $ isNothing (validatePrettyToJSON v)
+
+testRoundTripByteString ::
```

Comment:
We used this in an intermediate state of the PR, but I vote for keeping it here anyway so we won't have to find it next time we'll need it.